modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -78,8 +79,9 @@ function acquia_cms_common_install() {
   }
 
   // Install the syslog module.
-  $module_installer = \Drupal::service('module_installer');
   $module_installer->install(['syslog']);
```

Comment:
@apathak18 No need to call install method again below, we can update the same function i.e
```suggestion
  $module_installer->install(['syslog', 'purge_processor_lateruntime']);
```

Reply:
Also, I think we should do the same for purge_processor_cron & purge_ui module.
cc: @rajeshreeputra / @chandan-singh7929

Reply:
Done